src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -253,6 +253,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
           Failure(regrets)
         case success => success
       }
+      case _ => //nothing
```

Comment:
so: if it's done, then we don't need to abort it; do nothing. if it's not done but has no workflow id, then we can't abort it. what's happened if a workflow has got into this status? is this a thing we need to worry about?

at the very least i'd update the comment to something like "no need to abort workflows that are done, and can't abort workflows that don't have an id"